### PR TITLE
Adds inhand sprites to shoes.

### PR DESCRIPTION
**Pull request by alsoandanswer**
So, well, you can see yourself holding shoes.

SO MUCH IMMERSION WHOA

also actually replaces jackboots/unathi with jackboots/toeless properly

dreammaker_PB7apec40m
REMINDER : ITEM_STATE IS IN CHARGE OF INHANDS. WHEN IN DOUBT, DON'T TOUCH IT.
PLEASE.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/modules/clothing/shoes/colour.dm [hunk 1]
```diff
@@ -11,202 +12,217 @@
...
 /obj/item/clothing/shoes/flats
 	desc = "A pair of black, low-heeled women's flats."
 	name = "black dress flats"
 	icon = 'icons/obj/clothing/cheongsams.dmi'
-	icon_state = "dressflatsblack"
-	item_state = "dressflatsblack"
+	icon_state = "blackdf"
+	item_state = "blackdf"
 	species_restricted = null
 	contained_sprite = 1
```
Comment (fernerr) [inline code/modules/clothing/shoes/colour.dm]: As the flats are a contained sprite, the inhand sprites can't apply as it looks for the sprites in 'icons/obj/clothing/cheongsams.dmi'.

File: code/modules/clothing/shoes/leg_guard.dm [hunk 2]
```diff
@@ -2,6 +2,7 @@
 	name = "leg guards"
 	desc = "These will protect your legs and feet."
 	icon_state = "leg_guards_riot"
+	icon_state = "jackboots"
```
Comment (fernerr) [inline code/modules/clothing/shoes/leg_guard.dm]:
```suggestion
	item_state = "jackboots"
```

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 3]
```diff
@@ -14,11 +14,13 @@
 /obj/item/clothing/shoes/mime
 	name = "mime shoes"
 	icon_state = "white"
+	item_state = "white"
 
 /obj/item/clothing/shoes/swat
 	name = "\improper SWAT shoes"
 	desc = "When you want to turn up the heat."
 	icon_state = "swat"
+	icon_state = "swat"
```
Comment (fernerr) [inline code/modules/clothing/shoes/miscellaneous.dm]:
```suggestion
	item_state = "swat"
```

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 4]
```diff
@@ -32,6 +34,7 @@
 	name = "combat boots"
 	desc = "When you REALLY want to turn up the heat."
 	icon_state = "jungle"
+	icon_state = "jungle"
```
Comment (fernerr) [inline code/modules/clothing/shoes/miscellaneous.dm]:
```suggestion
	item_state = "jungle"
```

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 5]
```diff
@@ -54,18 +57,20 @@
...
 	name = "magic shoes"
 	icon_state = "black"
+	item_state = "black"
 	body_parts_covered = FEET
 
 /obj/item/clothing/shoes/sandal/flipflop
 	name = "flip flops"
 	desc = "A pair of foam flip flops. For those not afraid to show a little ankle."
 	icon_state = "thongsandal"
+	icon_state = "thongsandal"
```
Comment (fernerr) [inline code/modules/clothing/shoes/miscellaneous.dm]:
```suggestion
	item_state = "thongsandal"
```

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 6]
```diff
@@ -54,18 +57,20 @@
...
 
 /obj/item/clothing/shoes/clown_shoes
 	desc = "The prankster's standard-issue clowning shoes. Damn they're huge!"
 	name = "clown shoes"
 	icon_state = "clown"
-	item_state = "clown_shoes"
+	item_state = "clown"
```
Comment (fernerr) [inline code/modules/clothing/shoes/miscellaneous.dm]:
```suggestion
	item_state = "clown_shoes"
```

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 7]
```diff
@@ -174,6 +184,7 @@
 	name = "high heels"
 	desc = "A pair of high-heeled shoes. Fancy!"
 	icon_state = "heels"
+	icon_state = "thongsandal"
```
Comment (fernerr) [inline code/modules/clothing/shoes/miscellaneous.dm]:
```suggestion
	item_state = "thongsandal"
```

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**